### ISSUE TITLE
Loading Satellite layer in Edge44 throws CSP14312 error

### ISSUE THREAD

**Issue by maeneak** [mapbox/mapbox-gl-js#7726]
**mapbox-gl-js version**: Any

**browser**: Edge 44 (html18.305)

### Steps to Trigger Behavior

 1. Navigate to any satellite styled MapBox map (including MapBox Studio)

### Link to Demonstration

![image](https://user-images.githubusercontent.com/20371477/50418557-bf973900-0877-11e9-8384-1ba622564482.png)

### Expected Behavior
Expect satellite raster maps to be displayed

### Actual Behavior
Images don't display and below error is indicated in console

```
CSP14312: Resource violated directive 'connect-src
        'self'
        https://www.mapbox.com
        https://*.tiles.mapbox.com
        https://api.mapbox.com
        https://tilestream-tilesets-staging.s3.amazonaws.com
        https://tilestream-tilesets-production.s3.amazonaws.com
        https://122e4e-mapbox.global.ssl.fastly.net
        https://cloudfront-staging.tilestream.net
        https://api-maps-staging.tilestream.net
        https://atlas-router
        https://dynamodb.us-east-1.amazonaws.com
        https://api.segment.io
        http://api.segment.io
        https://117-nxk-490.mktoresp.com
        http://117-nxk-490.mktoresp.com
        https://sentry.io
        ws://*:8080
        http://localhost:*' in <meta http-equiv="Content-Security-Policy">: https://events.mapbox.com/events/v2?access_token=tk.eyJ1IjoiY3JpdGljYWxtYXNzIiwiZXhwIjoxNTQ1NzMxMjAyLCJpYXQiOjE1NDU3Mjc2MDIsInNjb3BlcyI6WyJlc3NlbnRpYWxzIiwic2NvcGVzOmxpc3QiLCJtYXA6cmVhZCIsIm1hcDp3cml0ZSIsInVzZXI6cmVhZCIsInVzZXI6d3JpdGUiLCJ1cGxvYWRzOnJlYWQiLCJ1cGxvYWRzOmxpc3QiLCJ1cGxvYWRzOndyaXRlIiwic3R5bGVzOnRpbGVzIiwic3R5bGVzOnJlYWQiLCJmb250czpyZWFkIiwic3R5bGVzOndyaXRlIiwic3R5bGVzOmxpc3QiLCJ0b2tlbnM6cmVhZCIsInRva2Vuczp3cml0ZSIsImRhdGFzZXRzOmxpc3QiLCJkYXRhc2V0czpyZWFkIiwiZGF0YXNldHM6d3JpdGUiLCJ0aWxlc2V0czpsaXN0IiwidGlsZXNldHM6cmVhZCIsInRpbGVzZXRzOndyaXRlIiwic3R5bGVzOmRyYWZ0IiwiZm9udHM6bGlzdCIsImZvbnRzOndyaXRlIiwiZm9udHM6bWV0YWRhdGEiLCJkYXRhc2V0czpzdHVkaW8iLCJjdXN0b21lcnM6d3JpdGUiLCJhbmFseXRpY3M6cmVhZCJdLCJjbGllbnQiOiJtYXBib3guY29tIiwibGwiOjE1NDU3MjYwODEzNzAsIml1IjpudWxsfQ.L0KdWc9Enzp8RMBp970fAA. Resource will be blocked.
```

**Comment by maeneak** [comment]
I should add this issue does not reproduce on any other browser (including edge mobile) and have confirmed the same results on fresh VM install of windows 10.

**Comment by mourner** [comment]
Is this failing with the latest Mapbox GL JS, v0.52, too? I wonder if this is related to #7671 which just got fixed.

**Comment by ansis** [comment]
I was not able to reproduce this with 0.52.0 and https://www.mapbox.com/mapbox-gl-js/example/satellite-map/ in Edge 44.18262 and EdgeHTML 18.18262.

@maeneak can you confirm that https://www.mapbox.com/mapbox-gl-js/example/satellite-map/ works for you?

I think the embed page might not be on 0.52 yet. I'll work on getting that updated.

**Comment by maeneak** [comment]
Sorry guys been off on other projects. Can confirm this is fixed in 0.52.0. Have updated my dependencies and working great with react-mapbox-gl@4.2.0. Thanks for your help guys.
